.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+
+Thanks for contributing to cartopy!
+Please use this template as a guide to streamline the pull request you are about to make.
+
+Remember: it is significantly easier to merge small pull requests. Consider whether this pull
+request could be broken into smaller parts before submitting.
+
+-->
+
+
+
+## Rationale
+
+<!-- Please provide detail as to *why* you are making this change. -->
+
+
+## Implications
+
+<!-- If applicable, to the best of your knowledge, what are the implications of this change? -->
+
+
+<!--
+## Checklist
+
+ * If you have not already done so, ensure you've read and signed the Contributor Licence Agreement (CLA).
+   (See the [governance page](http://scitools.org.uk/governance.html) for the CLA and what to do with it).
+
+ * If this is a new feature, please provide an example of its use in the description. We may want to make a
+   follow-on pull request to put the example in the gallery!
+
+ * Ensure there is a suitable item in the cartopy test suite for the change you are proposing. 
```

Comment:
These could be tick-boxes?

Reply:
I don't really think they need to be (they are actually hidden when submitting because of the ``<!-- hidden -->``...

Reply:
Fair enough! Happy to leave these as they are.